src/shm_main.c

```diff
@@ -1026,7 +1026,7 @@ sr_shmmain_allocate_conn(sr_conn_ctx_t *conn)
     cid = ATOMIC_INC_RELAXED(main_shm->new_sr_cid);
     err_info = sr_shmmain_check_conn_lock(cid, &alive);
     if (err_info || alive || (cid == 0)) {
-        SR_LOG_WRN("Could not allocate connection ID %ld (%d)", cid, alive);
+        SR_LOG_WRN("Could not allocate connection ID %"PRIu32" (%ld)", cid, alive);
```

Comment:
Why are you printing `int` with `%ld`? `%d` should be fine.

Reply:
Fixed